patch is in the development version and allows to compile with recent curl
see https://trac.xiph.org/ticket/1097
diff -Naur vorbis-tools-1.1.1/ogg123/http_transport.c vorbis-tools-1.1.1.new/ogg123/http_transport.c
--- vorbis-tools-1.1.1/ogg123/http_transport.c	2005-06-13 09:11:44.000000000 -0400
+++ vorbis-tools-1.1.1.new/ogg123/http_transport.c	2007-11-23 22:02:26.000000000 -0500
@@ -116,7 +116,9 @@
   if (inputOpts.ProxyTunnel)
     curl_easy_setopt (handle, CURLOPT_HTTPPROXYTUNNEL, inputOpts.ProxyTunnel);
   */
+#ifdef CURLOPT_MUTE
   curl_easy_setopt(handle, CURLOPT_MUTE, 1);
+#endif
   curl_easy_setopt(handle, CURLOPT_ERRORBUFFER, private->error);
   curl_easy_setopt(handle, CURLOPT_PROGRESSFUNCTION, progress_callback);
   curl_easy_setopt(handle, CURLOPT_PROGRESSDATA, private);
